lib/ansible/modules/network/exos/exos_lldp.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+#
+# (c) 2019 Extreme Networks Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: exos_lldp
+version_added: "2.9"
+author: "Ruturaj Vyawahare (@ruturajvy)"
+short_description: Manage LLDP configuration on Extreme Networks EXOS network devices.
+description:
+  - This module provides declarative management of LLDP service
```

Comment:
Could you please add a note stating which connection type this module supports and add a link to the platform guide?